### PR TITLE
Dom layouting

### PR DESCRIPTION
**Pull request by terazus**

### CODE REVIEW

**Comment by terazus** [comment]
Live preview available here: https://euphonious-phoenix-621146.netlify.app

File: _static/css/custom.css [hunk 1]
```diff
@@ -1130,4 +1125,36 @@ div.topbar {
...
+    right: 35px;
+    padding-bottom: 10px;
+}
+
+.bd-search .icon {
+    top:27px;
+}
+
+input.rounded {
+    border-radius: 50%;
```
Comment (proccaserra) [inline _static/css/custom.css:1144]: border-radius: 5%

**Review by proccaserra — APPROVED** [review]

File: _static/css/custom.css (hunk 1 above)
Comment (proccaserra) [reply _static/css/custom.css:1144]: great stuff!

Comment (terazus) [reply _static/css/custom.css:1144]: Done, was the wrong selector. Set to 10px.

**Pull request merged**